InternetHospitalWebUI/src/app/Components/Layout/header/header.component.html

```diff
@@ -1,5 +1,18 @@
-<mat-toolbar *ngIf="!authenticationService.isauthorised()">
-    <button   mat-button [routerLink]="['/sign-up']">Sign Up</button>
-    <button   mat-button [routerLink]="['/sign-in']">Sign In</button>    
+<mat-toolbar color="primary">
+    <button mat-button [routerLink]="['/']">HOME</button>
+    <span class="example-fill-remaining-space"></span>      
+    <button  *ngIf="!(isLoggedIn | async)" mat-button [routerLink]="['/sign-up']">Sign Up</button>
+    <button  *ngIf="!(isLoggedIn | async)" mat-button [routerLink]="['/sign-in']">Sign In</button>   
+    <button *ngIf="(isLoggedIn | async)"  mat-button [matMenuTriggerFor]="appMenu">           
+        <img  src="https://blog.circle.com/content/images/2018/04/rachel.png" style="max-width: 30px;"/>
```

Comment:
Why this image. It would be better to put just our def picture for avatar (if we have it).